lib/present_page_to_publishing_api.rb

```diff
@@ -5,4 +5,10 @@ def publish(presenter_class)
     Services.publishing_api.patch_links(payload.content_id, links: payload.links)
     Services.publishing_api.publish(payload.content_id, nil, locale: payload.content[:locale])
   end
+
+  def save_draft(presenter_class)
+    payload = presenter_class.new
+    Services.publishing_api.put_content(payload.content_id, payload.content)
+    Services.publishing_api.patch_links(payload.content_id, links: payload.links)
```

Comment:
Patching the links makes these appear in the live content item as well as the draft.  This could mean a list of the draft ministers can be obtained by accessing the page's content item, even if reshuffle mode is activated.

The only way around this would be to switch the page to use edition links (but then they aren't recursive, so wouldn't work here).

Reply:
Correct. We're also aware folks could go and look at individual people's pages and see role appointments etc being applied, even while in reshuffle mode. I'll make sure it's raised tomorrow so everyone is aware, but I think it's a known and accepted thing.